server.js

```diff
@@ -1,15 +1,22 @@
 const express = require('express');
 const app = express();
-const port = process.env.PUBLIC_PORT || 5000;
+const port =5000;
+const mongoose = require('mongoose');
 
-app.get("/ping",(req,res)=>{
-  res.send("pong")
+const uri= 'mongodb+srv://anushka_poonia:anushkapoonia3777@cluster0.rniykvz.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Avoid hardcoding sensitive information like database URIs directly in your code. Use environment variables instead. [important]